index.html

```diff
@@ -41,4 +41,140 @@ <h2 class="h2_title">I am a Software Developer.</h2>
+      <li class="second_project_skills flex">html</li>
+      <li class="second_project_skills flex">Bootstrap</li>
+      <li class="second_project_skills flex">Ruby</li>
+    </ul>
+    <button class="see_project_works2">See Project</button>
+  </div>
+
+  <div class="third_project">
+    <h3 class="third_project_title">Profesional Art Printing Data</h3>
+    <p class="third_project_content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+    <ul class="third_project_skill_list flex">
+      <li class="third_project_skills flex">html</li>
+      <li class="third_project_skills flex">Bootstrap</li>
+      <li class="third_project_skills flex">Ruby</li>
+    </ul>
+    <button class="see_project_works3">See Project</button>
+  </div>
+  </div>
+
+  
+  <div class="recent_work_container2 flex">
+<div class="first_project">
+    <h3 class="first_project_title">Profesional Art Printing Data</h3>
+    <p class="first_project_content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+    <ul class="first_project_skill_list flex">
+      <li class="first_project_skills">html</li>
+      <li class="first_project_skills">Bootstrap</li>
+      <li class="first_project_skills">Ruby</li>
+    </ul>
+    <button class="see_project_works">See Project</button>
+  </div>
+
+  <div class="second_project">
+    <h3 class="second_project_title">Profesional Art Printing Data</h3>
+    <p class="second_project_content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+    <ul class="second_project_skill_list flex">
+      <li class="second_project_skills">html</li>
+      <li class="second_project_skills">Bootstrap</li>
+      <li class="second_project_skills">Ruby</li>
+    </ul>
+    <button class="see_project_works2">See Project</button>
+  </div>
+
+  <div class="third_project">
+    <h3 class="third_project_title">Profesional Art Printing Data</h3>
+    <p class="third_project_content">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+    <ul class="third_project_skill_list flex">
+      <li class="third_project_skills">html</li>
+      <li class="third_project_skills">Bootstrap</li>
+      <li class="third_project_skills">Ruby</li>
+    </ul>
+    <button class="see_project_works3">See Project</button>
+  </div>
+  </div>
+
+ </section>
+
+ <section class="About_me">
+  <div class="about_me_container">
+    <h2 class="about_me_title">About me.</h2>
+    <p class="about_me_content">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+    <button class="get_resume flex">Get My Resume</button>
```

Comment:
- [ ] It looks like you have not removed the border from all the `See Projects` buttons. Please kindly fix the button design to make it similar to the Figma template. Please try to stick with your selected design template for better UI and consistency. 🙂